lib/models/LifecycleConfiguration.js

```diff
@@ -672,9 +672,15 @@ class LifecycleConfiguration {
                     return Tag;
                 }).join('');
             }
-            const Filter = ((rulePrefix && tags) || (tags && tags.length > 1)) ?
-                `<Filter><And>${Prefix}${tagXML}</And></Filter>` :
-                `<Filter>${Prefix}${tagXML}</Filter>`;
+            let Filter;
+            if (rulePrefix && !tags) {
+                Filter = Prefix;
```

Comment:
Does this fix because the prefix was not included in the previous case? Asking because before I see `${Prefix}` always part of the `Filter` output in the previous code.

Reply:
It fixes it because before `${Prefix}` was always within the `<Filter>` xml tag, but the cli doesn't support Filter, so it wasn't parsing the Prefix from it

Reply:
:+1: